tutorials/viewer.md

```diff
@@ -1,58 +1,296 @@
+
+These properties can also be changed within the console as follows:
 
+```python
+viewer.layers['camera'].name = 'photographer'
+viewer.layers['astronaut'].visible = False
+```
+
+and these changes will instantly propagate to the gui. For more information about the different properties for different layer types checkout our layer specific tutorials listed below.
+
+### layer control panel
+
+On the lefthand side of the main canvas is a thin vertical control panel that is used to display more layer specific controls. To get a sense of it's full usage we'll start a new example with both `Image` and `Points` layers, where a `Points` layer is used to display 2D markers at different locations.
+
+```python
+%gui qt5
+from skimage import data
+import numpy as np
+import napari
+
+viewer = napari.Viewer()
+viewer.add_image(data.moon(), name=moon)
+viewer.add_points(512 * np.random.random((20, 2)), name=spots, size=20, face_color='red')
+```
+This example has added 20 red markers to a `Points` layer which we have named `spots`. You'll note that as we select the two different layers different interactive elements appear in the layer control panel. When we have the `moon` selected then a slider appears, when we have `spots` selected four different buttons appear, and when we have neither or both layers selected no elements appear. These elements allow you to control global properties of the layer like the contrast slider for an `Image` layer, or allow you to add and select points for a `Points` layer. For more information about the different elements in the control panel for different layer types checkout our layer specific tutorials listed below.
+
+Here is a demo of interactivity for the `Image` and `Points` layers in the example above.
+
+![image](resources/layer_control_panel.gif)
+
+### new layer buttons
+New `Points`, `Shapes`, and `Labels` layers can be added to the viewer using the new layer buttons in the bottom righthand corner of the gui. These correspond to the calls
+
+```python
+viewer.add_points(data)
+viewer.add_shapes(data)
+viewer.add_labels(data)
+```
+but with empty data. Once added in the gui these layers become accessible in the layers list and at `viewer.layers`.
+
+Layers can also be deleted by selecting them and the clicking on the trash icon, the far right button, or by dragging the layers and dropping them into the trash.
+
+In the console a layer at index `i` can be removed by
+```python
+viewer.layers.pop(i)
+```
+
+Adding and deleting layers from the gui looks as follows
+
+![image](resources/layer_new.gif)
+
+Notice here how we added two `Points` layers and used the layer properties widget to change their colors. We then added a `Shapes` layer which allowed us to draw some rectangles, before we deleted all our new layers.
+
+### status bar
+
+At the very bottom of the gui there is a status bar that contains useful updates and tips.
```

Comment:
Haha this makes me want to put random easter eggs in there, like "Check out Torchy's Tacos when you're at the annual SciPy conference!"

Reply:
;-)